### PR TITLE
Closes #5 - Create user model

### LINKED ISSUE #5: Create User Model
Fields: Name, email, password, isAdmin
Note: Validate email, and unique
Default: isAdmin - false
Require name, email, password

### PR DESCRIPTION
**Pull request by earthtone0ne**
Closes #5 

Completed user model & tests.

### CODE REVIEW

File: tests/server/models/user-test.js [hunk 1]
```diff
@@ -146,7 +146,60 @@ describe('User model', function () {
...
+
+
+        beforeEach(function () {
+            return User.create({ email: 'obama@gmail.com', password: 'potus', name: 'Mr. President'});
+        })
+
+
+        it('rejects a bad email',
+           function (){
+                // return User.create({email: 'jackalope@bob.com', password: 'passwrd', name: 'Bob'})
```
Comment (queerviolet) [inline tests/server/models/user-test.js]: Remove extraneous comment.

File: tests/server/models/user-test.js [hunk 2]
```diff
@@ -146,7 +146,60 @@ describe('User model', function () {
...
+                return buildBadEmail.validate()
+                .then(function(response){
+                    // expect(response.id).to.be.undefined;
```
Comment (queerviolet) [inline tests/server/models/user-test.js]: Same.

File: tests/server/models/user-test.js [hunk 3]
```diff
@@ -146,7 +146,60 @@ describe('User model', function () {
                 });
             });
         });
+   });
+
+    // Our tests below:
```
Comment (queerviolet) [inline tests/server/models/user-test.js]: Everything the light touches is your queendom. (you don't need this comment).

File: tests/server/routes/members-only-test.js [hunk 4]
```diff
@@ -7,7 +7,7 @@ var db = require('../../../server/db');
 
 var supertest = require('supertest');
 
-describe('Members Route', function () {
+xdescribe('Members Route', function () {
```
Comment (queerviolet) [inline tests/server/routes/members-only-test.js]: Why?

Comment (earthtone0ne) [inline tests/server/routes/members-only-test.js]: This test was already there. It isn't relevant yet, but I think we'll need it later, so I marked it pending.

Comment (queerviolet) [inline tests/server/routes/members-only-test.js]: Ok. Was it failing? It's not worth blocking the PR for, but in general, I'd only x-out an existing test if it's failing.

**Pull request merged**